crates/linker/src/generate_dylib/macho.rs

```diff
@@ -98,3 +102,23 @@ pub fn create_dylib_macho(
 
     Ok(std::fs::read(dummy_lib_file).expect("Failed to load dummy library"))
 }
+
+fn get_macos_version() -> String {
+    let mut cmd = Command::new("sw_vers");
+    cmd.arg("-productVersion");
+
+    let cmd_stdout = cmd
+        .output()
+        .expect("Failed to execute command 'sw_vers -productVersion'")
+        .stdout;
+
+    let full_version_string = String::from_utf8(cmd_stdout)
+        .expect("Failed to convert output of command 'sw_vers -productVersion' into a utf8 string");
+
+    full_version_string
+        .trim_end()
+        .split('.')
+        .take(3)
+        .collect::<Vec<&str>>()
+        .join(".")
+}
```

Comment:
I copied the definition from `crates/compiler/build/src/link.rs:1120` but perhaps there is a crate that is already shared between these two crates and this helper could be put in there?

Reply:
Could put in in `roc_target` which would be a shared crate, but it doesn't necessarily fit there either. I think duplication is fine here.